Client.xcodeproj/project.pbxproj

```diff
@@ -12078,6 +12081,7 @@
 					"-ObjC",
 					"-lxml2",
 				);
+				OTHER_SWIFT_FLAGS_BASE = "-DMOZ_TARGET_$(TARGET_NAME:upper) -DNO_USER_WALLETS -DNO_SKUS -DNO_BRAVE_TODAY";
```

Comment:
why not placing it in base.xconfig or other xconfig file?

Reply:
Good point, I completely forgot the others were in Base.xcconfig, will fix